SEED-platform/seed
# fix: update aws dependencies to support signature version 4

#### Any background context you want to provide?
AWS no longer supports signature version 3 with SES

#### What's this PR do?
Switches from `boto` to `boto3`, since `boto` doesn't support signature version 4.

#### How should this be manually tested?
1. `pip install -r requirements/aws.txt`
2. Verify that sending emails is still functional

#### What are the relevant tickets?
#2637
